listenbrainz/mbid_mapping_writer/mbid_mapper.py

```diff
@@ -86,7 +86,7 @@ def detune_query_string(self, query, is_artist_credit):
             after it is removed.
         """
 
-        strings = ["(", "[", " ft ", " ft. ", " feat ", " feat. ", " featuring ", " - "]
+        strings = ["(", "[", " ft ", " ft. ", " feat ", " feat. ", " featuring ", " - ", " & "]
         if is_artist_credit:
             strings.insert(0, ",")
```

Comment:
I think we only want to do this detuning for artist credits so it makes sense to to put it inside the if. otherwise seems fine,

Reply:
Ah good catch.